cli/azd/pkg/input/console.go

```diff
@@ -229,6 +229,7 @@ func (c *AskerConsole) StopSpinner(ctx context.Context, lastMessage string, form
 
 	c.spinner.StopMessage(lastMessage)
 	_ = c.spinner.Stop()
+	c.currentIndent = ""
```

Comment:
Is there a more concise rule on how "currentIndent" is handled by the `console`, that now can be documented on the member variable `console.currentIndent`?

Reply:
no. This was actually a bug. 
indentation should be reset whenever we stop a spinner.  Or it should come back to previous state, in case we support multi-levels of indentation.
But we are not yet in a multi-level (more than 2), so for now this is fine.